server/controllers/users.js

```diff
@@ -150,6 +151,41 @@ class UserController {
       next(error);
     }
   }
+
+  /**
+  * Get users and their corresponding profiles
+  * @async
+  * @param {object} req - Request object
+  * @param {object} res - Response object
+  * @param {object} next The next middleware
+  * @return {json} Returns json object
+  * @static
+  */
+  static async getUsers(req, res, next) {
+    try {
+      const users = await User.findAll({
+        include: [
+          {
+            model: models.Profile,
+            as: 'profile'
+          }
+        ]
+      });
+
+      if (isEmpty(users)) {
```

Comment:
an empty list of users is still a valid request.

This should not be here

Reply:
I note this feedback